bindings/wasm/package.json

```
@@ -22,9 +22,10 @@
     "serve": "webpack serve",
     "example:node": "node examples/dist/node.js",
     "example:browser": "http-server ./examples/dist/ -c-1 -o ",
-	  "example:account": "ts-node ./examples-account/src/node.ts",
-    "test": "npm run test:unit && concurrently -g --timings \"npm:test:node\" \"npm:test:browser:parallel\" \"npm:test:readme\"",
+    "example:account": "ts-node ./examples-account/src/node.ts",
+    "test": "npm run test:unit && concurrently -g --timings \"npm:test:node\" \"npm:test:account:node\" \"npm:test:browser:parallel\" \"npm:test:readme\"",
     "test:node": "mocha ./examples/dist/tests/node/*.js --parallel --jobs 4 --retries 3 --exit",
+    "test:account:node": "ts-mocha ./examples-account/src/tests/*.ts --parallel --jobs 4 --retries 3 --exit",
```

Specify timeout (applies per test unless I'm mistaken).

```suggestion
    "test:account:node": "ts-mocha ./examples-account/src/tests/*.ts --parallel --jobs 4 --retries 3 --timeout 180000 --exit",
```